README.md

```diff
@@ -18,7 +18,65 @@ with Bazel.
 
 ## Setup
 
-To be documented.
+### Using npm and Node.js rules
```

Comment:
Feels like the README could address the tradeoffs between using these rules vs. just using postcss binary directly (I'll add a postcss example to https://github.com/bazelbuild/rules_nodejs/tree/1.x/examples/app/styles)

Reply:
This makes me concerned whether taking `@bazel/postcss` might be breaking expectations of other users of packages generated from rules_nodejs - what do you think of e.g. forwarding the `npm_package_bin`-generated rule in addition to providing the `postcss_{binary,plugin}` rules

Reply:
I don't think there's a problem with one npm package exposing some custom rules around some tool, while the npm package for that tool can also be used standalone. We already have the same situation with `@bazel/typescript` (the g3 toolchain) vs just `typescript` which you can use straight from MS. For TS I documented tradeoffs https://github.com/bazelbuild/rules_nodejs/blob/master/docs/TypeScript.md#alternatives
so I'm just imagining a couple sentences similarly explaining what are the choices and why to pick one or the other.

Reply:
Thanks! Added a TODO to the readme